templates/Web/Projects/VueDefault/README_postaction.md

```diff
@@ -0,0 +1,56 @@
+﻿## Getting Started
+
+In the root directory of the project...
+
+1. Install node modules `yarn install` or `npm install`.
+2. Start development server `yarn start` or `npm start`.
+
+## Next Steps
+//{[{
```

Comment:
Everything above this symbol is used for context, the //{[{ symbol is used to define the start of an insertion. anything between the //{[{ and //}]} symbols will be inserted between the context provided (the context is above //{[{ and below //}]}).